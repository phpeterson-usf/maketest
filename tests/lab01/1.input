$(TESTS_DIR)/form.txt $(repo_dir)/complete.txt FIRST Phil LAST Peterson CITY Saratoga
