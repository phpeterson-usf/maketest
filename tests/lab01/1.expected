My name is Phil Peterson
I live in Saratoga
